android/res/values/styles-text.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
   <style name="MwmTextAppearance">
-    <item name="android:fontFamily" tools:ignore="NewApi">@string/robotoRegular</item>
+    <item name="android:fontFamily"
```

Comment:
Is there a real need to add these newlines?
Please, roll back to old newlines and spacing, not only here, but in all files modified by this pull request.